src/Docker.PowerShell/Cmdlets/InvokeContainerImage.cs

```diff
@@ -77,7 +78,7 @@ public class InvokeContainerImage : CreateContainerCmdlet
                     }
                     else if (PassThru.ToBool())
                     {
-                        WriteObject(await ContainerOperations.GetContainerById(createResult.ID, DkrClient));
+                        WriteObject((await ContainerOperations.GetContainersById(createResult.ID, DkrClient)).Single());
```

Comment:
.Single() will return a pretty odd error if it either isnt found or has more than 1. Should we throw a better error?